### PR TITLE
Dequeue

### PR DESCRIPTION
**Pull request by mattiaerre**
# Description

as per #12 this would be the `Stacks` and `Queue` coding challenge

### CODE REVIEW

**Comment by mattiaerre** [comment]
It'd be nice to have a review of this PR from you @scottlepich

File: package.json [hunk 1]
```diff
@@ -36,6 +38,7 @@
     "eslint-plugin-import": "^2.7.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-react": "^7.5.1",
+    "jest": "^23.1.0",
```
Comment (scottlepich) [inline package.json]: Rad

File: src/queue.test.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import Queue from './queue';
+
+describe('Queue', () => {
```
Comment (scottlepich) [inline src/queue.test.js]: I really like the process of using test first.  are you going to use this today?

Comment (mattiaerre) [reply src/queue.test.js]: thanks for merging my friend, nope I am not. I will ask more API/arch questions; feel free to use it if you like 😉

**Pull request merged**